### PR TITLE
Fix/chat 508 : get users/spaces avatars from Social REST API instead of Chat REST API

### PR DESCRIPTION
**Pull request by MrHanachoo**
This PR changes users/spaces avatars urls:
- user: 

/rest/chat/api/1.0/user/getAvatarURL/USER_ID => /rest/v1/social/users/USER_ID/avatar
- space: 

/rest/chat/api/1.0/user/getSpaceAvartar/SPACE_ID  => /rest/v1/social/spaces/SPACE_ID/avatar

### CODE REVIEW

File: application/src/main/webapp/js/chat.js [hunk 1]
```diff
@@ -3617,7 +3617,7 @@ ChatApplication.prototype.renderRoomUser = function(user, showOfflineUsers) {
     }
     html += ">";
     html += "  <div class='msUserAvatar pull-left'>";
-    html += "    <span class='msAvatarLink avatarCircle'><img onerror='this.src=\'/chat/img/user-default.jpg\'' src='/" + eXo.env.portal.rest +"/chat/api/1.0/user/getAvatarURL/" + user.name + "' alt='" + user.fullname + "'></span>";
+    html += "    <span class='msAvatarLink avatarCircle'><img onerror='this.src=\'/chat/img/user-default.jpg\'' src='/" + eXo.env.portal.rest +"/v1/social/users/" + user.name + "/avatar' alt='" + user.fullname + "'></span>";
```
Comment (MrHanachoo) [inline application/src/main/webapp/js/chat.js]: Fixed.

File: exo-addons-chat-extension-config-community/src/main/java/org/exoplatform/addons/chat/api/UserRestService.java [hunk 2]
```diff
@@ -40,46 +40,7 @@ public UserRestService()
   {
   }
 
-  @GET
```
Comment (boubaker) [inline exo-addons-chat-extension-config-community/src/main/java/org/exoplatform/addons/chat/api/UserRestService.java]: Shouldn't we deprecate the API changes in this version first and delete it only in next versions ?

File: exo-addons-chat-extension-config/src/main/java/org/exoplatform/addons/chat/api/UserRestService.java [hunk 3]
```diff
@@ -72,57 +72,5 @@ public Response getToken(@QueryParam("tokenOnly") String tokenOnly) throws Excep
             .build();
   }
   
-  @GET
```
Comment (boubaker) [inline exo-addons-chat-extension-config/src/main/java/org/exoplatform/addons/chat/api/UserRestService.java]: Shouldn't we deprecate the API changes in this version first and delete it only in next versions ?

File: exo-addons-chat-extension-config-community/src/main/java/org/exoplatform/addons/chat/api/UserRestService.java (hunk 2 above)
Comment (thomasdelhomenie) [inline exo-addons-chat-extension-config-community/src/main/java/org/exoplatform/addons/chat/api/UserRestService.java]: This endpoint does not work anymore (the avatar is not anymore in the JCR), so tagging it as deprecated would require to update it to make it work with the new storage (not a big work, but it needs to be done). Also there is no documentation on these endpoints so I think we can consider them as experimental APIs and remove them directly. WDYT ?

Comment (boubaker) [inline exo-addons-chat-extension-config-community/src/main/java/org/exoplatform/addons/chat/api/UserRestService.java]: ok for me. thx

**Comment by MrHanachoo** [comment]
merged into exodev/develop

**Pull request closed without merging**